packaging/desktop/package_darwin.sh

```diff
@@ -206,28 +214,32 @@ save() {
   cd $out_dir
   if [ "$save_dir" = "" ]; then
     echo "Saved files to $out_dir"
-  else
-    mkdir -p $save_dir
-    cd $save_dir
-    platform_dir="$save_dir/$platform"
-    echo "Saving files to $platform_dir"
-    # DMG
-    mkdir -p $platform_dir
-    mv $out_dir/$dmg_name $platform_dir
-    # Zip
-    mkdir -p "$platform_dir-updates"
-    mv $out_dir/$zip_name "$platform_dir-updates"
-    # Sourcemap
-    mkdir -p "$save_dir/electron-sourcemaps"
-    mv "$out_dir/$sourcemap_name" "$save_dir/electron-sourcemaps"
+    return
   fi
+  mkdir -p $save_dir
+  cd $save_dir
+  platform_dir="$save_dir/$platform"
+  echo "Saving files to $platform_dir"
+  # DMG
+  mkdir -p "$platform_dir"
+  mv "$out_dir/$dmg_name" "$platform_dir"
+  # Zip
+  mkdir -p "$platform_dir-updates"
+  mv "$out_dir/$zip_name" "$platform_dir-updates"
+  # Sourcemap
+  mkdir -p "$save_dir/electron-sourcemaps"
+  mv "$out_dir/$sourcemap_name" "$save_dir/electron-sourcemaps"
+  # Support files
+  mkdir -p "$platform_dir-support"
+  mv "$out_dir/$update_json_name" "$platform_dir-support"
+}
 
-  if [ ! "$s3host" = "" ]; then
-    keybase sign -d -i "$platform-updates/$zip_name" -o $sig_path
-    $release_bin update-json --version=$app_version --src="$platform-updates/$zip_name" --uri="$s3host/$platform-updates" --signature="$sig_path" > update-$platform-$run_mode.json
+s3sync() {
+  if [ ! "$bucket_name" = "" ] && [ ! "$save_dir" = "" ]; then
+    s3cmd sync --acl-public --disable-multipart $save_dir/* s3://$bucket_name/
```

Comment:
Quoting `$save_dir` is still possible if you leave the `*` outside, something like `"$save_dir"/*`.